src/lib/ContractInfo.ts

```diff
@@ -11,6 +12,17 @@ export class ContractInfoHelper {
             default: throw new Error('Unknown contract type: ' + o.type);
         }
     }
+
+    public static objectToContractType(o: ContractInfoEntry): VestingContract | HashedTimeLockedContract {
+        switch (o.type) {
+            case 1 /* Nimiq.Account.Type.VESTING */:
```

Comment:
Why does this not use `Nimiq.Account.Type.VESTING` and `Nimiq.Account.Type.HTLC`?

Reply:
Because Nimiq lib is not loaded in iframe.

Reply:
Although I think the compiler actually compiles the variable out and leaves the number in, because it's shorter.

Reply:
Correct for 157. The others should not be executed, as the class is only used as static in the iframe. But maybe that just makes my point in saying that the compiler removes the variables and only uses the numbers anyway, so should we maybe leave the variables in?

Reply:
If we can, we should use the constants. If we can not, then we should be consistent about it I think and use the values with the corresponding comments everywhere.

Reply:
We cannot use constants. They stay in the built output and throw a `Nimiq is not defined` error.

Reply:
I would then say remove the other occurrences of `Nimiq.*`, too.

Reply:
No, I say we should only use the values with commented constants where necessary, not automatically everywhere.

Reply:
Personally I find it somewhat strange to have both versions in the same file.

Reply:
I opened https://github.com/nimiq/accounts/issues/200 to track this problem.